contracts/CO30_Matias_BlackSmokeRising.blud_cordelia/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,13 @@
+Terrain in the AO is covered with thick jungles, making traversal difficult outside of the river and paths.
```

Comment:
Wrong usage of the section

Reply:
INSERT THINGS PLAYERS SHOULD KNOW ABOUT WHERE THEY ARE DEPLOYING AND/OR EVENTS THAT HAPPENED LEADING UP TO THEIR DEPLOYMENT OR TO THE GIVEN CONFLICT

Reply:
Literally something about where they are deploying

Reply:
Yeah I guess you're right, I had not realised we no longer have a 'terrain considerations' section, however, I think this is mostly intended for events and context, not really for descriptions of terrain alone, but they could be included.